packages/components/src/navigable-container/menu.tsx

```diff
@@ -44,6 +43,8 @@ export function NavigableMenu(
 			// in an offset.
 			return 0;
 		}
+
+		return undefined;
```

Comment:
This is pedantic, but it prevents a warning from `eventToOffset()` sometime not having a `return` value.